### PR TITLE
Update "Deploy Hyperlane with Local Agents" page 

### PR DESCRIPTION
**Pull request by ltyu**
- Updates with latest CLI commands
- Link from "Deploy Hyperlane to a new chain"
- Add initialize agent config section

### CODE REVIEW

**Review by jmrossy — APPROVED** [review]

File: docs/deploy-hyperlane.mdx [hunk 1]
```diff
@@ -174,3 +174,10 @@ Finally, push local registry to github fork and submit a PR. Please include a [c
 
 Thank you for contributing to the future of permissionless interop 🫡
 :::
+
+:::tip
+
+**Next Steps**
+
+If you want deploy a Relayer to relay messages to your chain, continue on to [Deploy Hyperlane with Local Agents](/guides/deploy-hyperlane-local-agents.mdx#4-run-a-relayer)
```
Comment (daniel-savu) [inline docs/deploy-hyperlane.mdx:182]: What's the ISM used in the core deployment? If it's the trusted / noop ISM a relayer is enough, but for a Multisig ISM a validator is needed as well

File: docs/partials/deploy-hyperlane/_deploy-contracts.mdx [hunk 2]
```diff
@@ -24,25 +24,20 @@ npm install -g @hyperlane-xyz/cli
 
 <CliChainsPartial />
 
-#### ISM
+#### Core deployment with Multisig ISM
 
 Now the CLI will know how to interact with all your chains, but it will also need to know how to configure your Interchain Security Module (ISM).
 
-To create a multisig ISM configs, you can define it manually using JSON or YAML (see [example config here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/v3/typescript/cli/examples/multisig-ism.yaml)), or create it with the following command:
+To initialize a core config with multisig ISM config, you can define it manually using JSON or YAML (see [example config here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/cli/examples/core-config.yaml)), or create it with the following command:
```
Comment (daniel-savu) [inline docs/partials/deploy-hyperlane/_deploy-contracts.mdx:31]: nit: I know the previous URL was also relative, but I think it's good practice to use permalinks instead so we always point to the same commit. This prevents having broken links when things get moved around

**Review by daniel-savu — APPROVED** [review]
LGTM, thanks 🤠

File: docs/deploy-hyperlane.mdx (hunk 1 above)
Comment (ltyu) [reply docs/deploy-hyperlane.mdx:182]: It's just a trusted relayer

**Pull request merged**